int32[] ys
int32[] xs
float32[4] probs
uint16 num_lanes
sensor_msgs/Image rawimg
sensor_msgs/Image laneimg